packages/ns-api/Makefile

```diff
@@ -24,6 +24,10 @@ define Package/ns-api
 	DEPENDS:=+python3-nethsec +python3-openssl +python3-urllib +python3-idna
 	PKGARCH:=all
 endef
+
+define Package/ns-api/conffiles
+/etc/config/ns-api
```

Comment:
Indent with a tab here

Reply:
Thought to add the tab too, [found out that is not possible](https://openwrt.org/docs/guide-developer/packages#packageconffiles_optional)

Reply:
Whoa! We should fix other packages!